docs/architecture/how/gas.md

```diff
@@ -46,6 +54,63 @@ instead of once at the start. However, remember that the execution may happen on
 a different shard than the signer account. Therefore we cannot access the
 signer's balance while executing.
 
+### Burning Gas
+
+Buying gas immediately removes a part of the signer's tokens from the total
+supply. However, the equivalent value in gas still exists in the form of the
+receipt and the unused gas will be converted back to tokens as a refund.
+
+The gas spent on execution on the other hand is burnt and removed from total
+supply forever. Unlike gas in other chains, none of it goes to validators. This
+is roughly equivalent to the base fee burning mechanism which Ethereum added in
+[EIP-1559](https://eips.ethereum.org/EIPS/eip-1559). But in Near Protocol, the
+entire fee is burnt because there is no [priority
+fee](https://ethereum.org/en/developers/docs/gas/#priority-fee) that Ethereum
+pays out to validators.
+
+The following diagram shows how gas flows through the execution of a
+transaction. The transaction consists of a function call performing a cross
+contract call, hence two function calls in sequence. (Note: This diagram is
+heavily simplified, more accurate diagrams are further down.)
+
+![Very Simplified Gas Flow Diagram](https://user-images.githubusercontent.com/6342444/265536127-5934362f-18d1-4653-8dc6-e930af57ebc5.svg)
```

Comment:
Typo: aline.near instead of alice.near
![image](https://github.com/near/nearcore/assets/86772482/c5068ce7-06e2-4e54-a884-cef2ab318325)

Reply:
Thanks! Somehow the :fire: also didn't display correctly. Should be fixed now.